sentry-core/Cargo.toml

```diff
@@ -40,9 +40,12 @@ uuid = { version = "1.0.0", features = ["v4", "serde"], optional = true }
 sys-info = { version = "0.9.1", optional = true }
 build_id = { version = "0.2.1", optional = true }
 findshlibs = { version = "=0.10.2", optional = true }
+rustc_version_runtime = { version = "0.2.1", optional = true }
+chrono = { version = "0.4.22",  features = ["serde"], optional = true}
 
 [target.'cfg(target_family = "unix")'.dependencies]
 pprof = { version = "0.10.1", optional = true, default-features = false}
+libc = { version = "^0.2.66", optional = true }
```

Comment:
(I am novice to Rust)

But if Rust is similar to C, then`libc` dependency is bad for portability. Consumers of such transitive dependency have to start thinking with which `libc` version executable was linked during compilation.

We are trying to avoid any Linux native sys dependencies and use Rust native alternatives (eg. `native-tls` -> `rustls`, etc). 

Maybe there are a Rust alternative to `libc` usage? If I understand it is used only to get threadId?

Reply:
> if you have a specific alternative in mind though, I'm happy to reconsider this.

No. Sadly I don't.